test/hotspot/jtreg/containers/docker/TestPids.java

```diff
@@ -93,6 +93,17 @@ private static void checkResult(List<String> lines, String lineMarker, String ex
                 String[] parts = line.split(":");
                 System.out.println("DEBUG: line = " + line);
                 System.out.println("DEBUG: parts.length = " + parts.length);
+                if (expectedValue.equals("no_value_expected")) {
+                    Asserts.assertEquals(parts.length, 2);
```

Comment:
Is "no_value_expected" generated by Docker? I searched the entire HotSpot source code and couldn't find it. I also couldn't find "WARNING: Your kernel does not support pids limit capabilities". 

To make it easier to understand this test, I would suggest grouping all messages that were generated outside of HotSpot into something like:

```
// These messages are generated by Docker
static final String warning_kernel_no_pids_support = "WARNING: Your kernel does not support pids limit capabilities";
static final String no_value_expected = "no_value_expected";
```

Reply:
Oh, I misunderstood the test. "no_value_expected" was passed in from `testPids()` in this file, but that's confusing because you are expecting an integer value. Maybe it should be "any_integer"?

Reply:
> Maybe it should be "any_integer"?

+1

Reply:
> Is "no_value_expected" generated by Docker? I searched the entire HotSpot source code and couldn't find it. I also couldn't find "WARNING: Your kernel does not support pids limit capabilities".
> 

Hi, this warning is showing up on some of our Linux ppc64le machines where the pids limit capabilities is not supported.

Best regards, Matthias

Reply:
Hello, I adjusted to any_integer, and introduced the "final String warning_kernel_no_pids_support" . I think the message is more related to the kernel features so I did not add the add the  'generated by Docker'  comment.

Best regards, Matthias